### PR TITLE
Bump versions to 1.21.1 and update changelogs

### PR DESCRIPTION
**Pull request by dcoutts**

### CODE REVIEW

File: cardano-node/ChangeLog.md [hunk 1]
```diff
@@ -651,9 +670,9 @@ No changes in the node. There were changes in the cardano-api and cardano-cli.
 ## 1.9.0 -- March 2020
 
 ### node changes
-- Remove CLI override for genesis file. It must be in the confg file. (#683)
-- Genesis file path in confg relative to the config file location (#648)
-- For security adjust default confg to not listen on local ports (#707)
+- Remove CLI override for genesis file. It must be in the config file. (#683)
+- Genesis file path in config relative to the config file location (#648)
```
Comment (disassembler) [inline cardano-node/ChangeLog.md:674]: Can we confirm that #648 doesn't cause any issues with absolute paths?

File: cardano-node/ChangeLog.md [hunk 2]
```diff
@@ -1,5 +1,24 @@
 # Changelog for cardano-node
 
+## 1.21.1 -- September 2020
+
+### node changes
+- Fix configuration defaults that was interfering with using systemd socket
+  activation (#1927)
+- Workaround for building on Linux systems without systemd (#1775)
+- Fix the severity level for some protocol tracers (#1910)
```
Comment (disassembler) [inline cardano-node/ChangeLog.md:9]: @jbgi can you confirm logs look fine after this change?

**Comment by tatyanavych** [comment]
We have not yet release 1.21.0. We are releasing one product increment after 1.20.0

File: cardano-node/ChangeLog.md (hunk 2 above)
Comment (jbgi) [reply cardano-node/ChangeLog.md:9]: looks good, no apparent change in logs.

File: cardano-node/ChangeLog.md (hunk 1 above)
Comment (dcoutts) [reply cardano-node/ChangeLog.md:674]: I didn't change anything here in the changelog except for spelling.

**Review by intricate — APPROVED** [review]

**Pull request merged**